README.md

```diff
@@ -210,3 +219,6 @@ un-listed error, simply execute `run.py` again with the same exact options.
 #### Cloudformation
 On occasion the Cloudformation stack will fail to delete properly. Simply go
 into the AWS console and find the Cloudformation stack and delete it.
+
+#### When all hope is lost
+Most likely it's because you have existing settings in your AWS region.  Pick another AWS region and start over. 
```

Comment:
This is not specific enough and should probably be in the troubleshooting section.